Shared/UI/Views/SpeechBubbleView.swift

```diff
@@ -33,13 +33,6 @@ class SpeechBubbleView: BaseView {
             guard let cgColor = self.bubbleLayer.fillColor else { return nil }
             return UIColor(cgColor: cgColor)
         }
-        set {
-            // CALayers are implicitly animated. Disable animations so the color changes immediately.
-            CATransaction.begin()
-            CATransaction.setDisableActions(true)
-            self.bubbleLayer.fillColor = newValue?.cgColor
-            CATransaction.commit()
-        }
```

Comment:
Previously this was the only way to set the bubble color and it prevented animations. I made a set function down below.